app/src/main/java/com/doubtless/doubtless/screens/profile/OtherUsersProfileViewModel.kt

```diff
@@ -0,0 +1,124 @@
+package com.doubtless.doubtless.screens.profile
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.viewModelScope
+import com.doubtless.doubtless.analytics.AnalyticsTracker
+import com.doubtless.doubtless.screens.auth.User
+import com.doubtless.doubtless.screens.dashboard.usecases.FetchUserDataUseCase
+import com.doubtless.doubtless.screens.home.entities.FeedEntity
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+class OtherUsersProfileViewModel(
```

Comment:
this does exactly the same thing as dashboard viewmodel then we need to create a separate one?